script.js

```diff
@@ -0,0 +1,177 @@
+},
+{
+  id: 4,
+  name: 'Autum Huffman',
+  profession: 'CAREER SERVICESS',
+  image: 'images/four.png',
+  discretion: 'Autumn is a reformed academic with a passion for democratizing education and helping students meet their potential. Prior to Microverse, she was a college instructor, education researcher, college & career coach, and helped bootstrap a Florida non-profit education program. In her spare time Autumn loves hiking, amateur photography, and having dance parties with her 4 year old.',
+},
+{
+  id: 5,
+  name: 'Bee Quesada',
+  profession: 'STUDENT SUCCESS',
+  image: 'images/Screenshot (21).png ',
+  discretion: 'Based in the heart of Central America, Bee is a language-loving nerd with a deep passion for education and a service mindset. Her background is in customer experience, people management, and learning & development. In her free time, Bee can be found reading post-apocalyptic literature and thrifting for vintage 90s fashion.',
+},
+];
+
+// const sponsor = [
+//   {
+//     name: 'SOMA CAPITAl',
+//     logo: 'sponsers-img/5e581622c52d562e19f15c2f_soma-capital.png',
+//   },
+//   {
+//     name: 'Y 50',
+//     logo: 'sponsers-img/5e5816210e511c21a03350d2_50Ylogo.png',
+//   },
+//   {
+//     name: 'GENERAL CATALYST',
+//     logo: 'sponsers-img/5e5816210e511cdd323350d3_gc.webp',
+//   },
+//   {
+//     name: 'yclogo',
+//     logo: 'sponsers-img/5e5816226dfe7b54f5cef482_yclogo.png',
+//   },
+//   {
+//     name: 'L2 VENTURES',
+//     logo: 'sponsers-img/5e5816226dfe7b896acef481_l2.png',
+//   },
+//   {
+//     name: 'hackvc',
+//     logo: 'sponsers-img/5e58162203fc89e9aa5acc3a_hackvc.webp',
+//   },
+//   {
+//     name: '',
+//     logo: '',
+//   },
+//   {
+//     name: 'Northzone',
+//     logo: 'sponsers-img/61522595be718075aca25e5c_northzone.png',
+//   },
+// ];
+
+// const learn = [{
+//   image: 'images/program_icon_04.png',
+//   title: 'NO UPFRONT COST',
+//   discretion: 'You owe us nothing until you get a job —no matter what country you’re in.',
+
+// }, {
+//   image: 'images/program_icon_03.png',
+//   title: 'COLLAB LEARNING',
+//   discretion: 'Learn through pair programming, group projects,and project reviews.',
+
+// }, {
+//   image: 'images/program_icon_01.png',
+//   title: 'GLOBAL CHANCE',
+//   discretion: 'Connect with global jobs after gaining experience working on a remote team.',
+// }, {
+//   image: 'images/program_icon_05.png',
+//   title: 'SUPPORT AND LIABLE',
+//
+// },
+// ];
+// const story = [{
+//   name:"kevin Mwangi",
+//   image:"images/Screenshot (17).png"
+//   ,link:"https://www.microverse.org/blog/a-career-with-microsoft-how-microverse-helped-pave-the-way"
+// },
+// {name:"Gabriela Cruz Olivera",
+// image:"images/Screenshot (11).png",
+// link:"https://www.microverse.org/blog/join-the-global-job-market-as-a-web-developer"}
```

Comment:
- [ ] I kindly suggest that your remove commented out code from your codebase to make your work neater.

Reply:
Fixed.